server/utils/placementCriteriaUtils.ts

```diff
@@ -46,7 +46,7 @@ export const placementCriteria: Record<PlacementCriteria, string> = {
   acceptsNonSexualChildOffenders: 'Non sexual offences against children',
   acceptsHateCrimeOffenders: 'Hate based offences',
   isWheelchairDesignated: 'Wheelchair accessible',
-  isSingleRoom: 'Single room',
+  isSingle: 'Single room',
```

Comment:
@pezholio I've just clocked we just sentence case these for the `response` method so `isSingle` will be 'is single' which could be misinterpreted. 
The only way around it is: 
- have an object with translations that need something more complex than just sentence casing 
- if we can't find the translation then just sentence case. 
Otherwise we could change `isSingle` to `isSingleRoom` in the API but I can imagine that will cause headaches elsewhere

Reply:
Ah, I've just seen the failing test. Gaah. I think translations would be better to be honest - even "Is Single Room" seems a bit clumsy.